x/nft/spec/02_messages.md

```diff
@@ -25,34 +25,34 @@ type MsgIssueDenom struct {
 
-| **Field** | **Type** | **Description**                                                                                                                  |
-| :-------- | :------- | :------------------------------------------------------------------------------------------------------------------------------- |
-| ID        | `string` | The unique ID of the NFT being transferred.                                                                                      |
-| DenomId   | `string` | The unique ID of the denomination, necessary as multiple denominations are able to be represented on each chain.                 |
-| Name      | `string` | The name of the NFT being transferred.                                                                                           |
-| URI       | `string` | The URI pointing to a JSON object that contains subsequent tokenData information off-chain                                       |
-| Data      | `string` | The data of the NFT                                                                                                              |
-| Sender    | `string` | The account address of the user sending the NFT. By default it is __not__ required that the sender is also the owner of the NFT. |
-| Recipient | `string` | The account address who will receive the NFT as a result of the transfer transaction.                                            |
+| **Field** | **Type** | **Description**                                                                                                  |
+| :-------- | :------- | :--------------------------------------------------------------------------------------------------------------- |
+| Id        | `string` | The unique ID of the NFT being transferred.                                                                      |
+| DenomId   | `string` | The unique ID of the denomination, necessary as multiple denominations are able to be represented on each chain. |
+| Sender    | `string` | The account address of the user sending the NFT.                                                                 |
+| Recipient | `string` | The account address who will receive the NFT as a result of the transfer transaction.                            |
 
 ```go
 // MsgTransferNFT defines an SDK message for transferring an NFT to recipient.
 type MsgTransferNFT struct {
     Id        string
     DenomId   string
-    Name      string
-    URI       string
-    Data      string
     Sender    string
     Recipient string
 }
 ```
 
 ## MsgEditNFT
 
-This message type allows the `TokenURI` to be updated. By default anyone can execute this Message type. **It is highly recommended that a custom handler is made to restrict use of this Message type to prevent unintended use.**
+This message type allows the `TokenURI` to be updated. `Sender` of this message should be the `Owner` of the NFT and
+`Creator` of the denomination corresponding to `DenomId`.
```

Comment:
So only time one can modify metadata is when creator of denomination is owner of the nft?

Reply:
Yes.